src/repositories/userPowerRepository.test.ts

```diff
@@ -103,4 +135,36 @@ function findUsersThatDidntSyncTheirPowerTestCases() {
     // usersWithoutPowerSync should not include user1
     assert.isNotOk(usersWithoutPowerSync.find(user => user.id === user1.id));
   });
+  it('should return users without a boosted any project', async () => {
```

Comment:
```suggestion
  it('should not return users without any boosted project', async () => {
```

Reply:
Just a text suggestion to make it clear, I was getting confused as the headline was not what was ocurring on the code execution.

Reply:
Thanks @CarlosQ96